app/services/spree_shopify_importer/data_parsers/stock_items/base_data.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+module SpreeShopifyImporter
+  module DataParsers
+    module StockItems
+      class BaseData
+        CONTINUE = 'continue'
+        def initialize(spree_stock_location, inventory_level)
+          @spree_stock_location = spree_stock_location
+          @inventory_level = inventory_level
+        end
+
+        def variant_data_feed
+          SpreeShopifyImporter::DataFeed
+            .where(shopify_object_type: 'ShopifyAPI::Variant')
+            .where('data_feed like ?', "%inventory_item_id\":#{@inventory_level.inventory_item_id}%").first
+        end
+
+        def variant
```

Comment:
I also think that this method should be private

Reply:
I don't think so, the saver use it